easybuild/easyblocks/g/gromacs.py

```diff
@@ -328,10 +328,11 @@ def configure_step(self):
 
             if get_software_root('imkl'):
                 # using MKL for FFT, so it will also be used for BLAS/LAPACK
-                self.cfg.update('configopts', '-DGMX_FFT_LIBRARY=mkl -DMKL_INCLUDE_DIR="$EBROOTMKL/mkl/include" ')
+                self.cfg.update('configopts', '-DGMX_FFT_LIBRARY=mkl -DMKL_INCLUDE_DIR="%s" ' %
+                                os.path.join(os.getenv('MKLROOT'), 'include'))
```

Comment:
Although the above works and is correct,
I still think that the correct way to do this from an easyblock perspective would be to do
```
imkl_root = get_software_root('imkl')
  if imkl_root:
    self.cfg.update('configopts', '-DGMX_FFT_LIBRARY=mkl -DMKL_INCLUDE_DIR="%s" ' % os.path.join(imkl_root, 'mkl', 'include'))
```

Reply:
Yeah, I agree it's probably better to avoid relying on `$MKLROOT`.
A bit cleaner:

```python
imkl_root = get_software_root('imkl')
if imkl_root:
    # using MKL for FFT, so it will also be used for BLAS/LAPACK
    imkl_include = os.path.join(imkl_root, 'mkl', 'include')
    self.cfg.update('configopts', '-DGMX_FFT_LIBRARY=mkl -DMKL_INCLUDE_DIR="%s" ' % imkl_include)

Reply:
OK, I'll change that.